integrationtest/docker/jdk8/start.sh

```diff
@@ -36,12 +36,19 @@ case "$COMMAND" in
     JAVA_OPTS="$JAVA_OPTS -Dgearpump.hostname=$(hostname)"
     sh "$SUT_HOME"/bin/"$COMMAND" "$@"
     ;;
+  storm-drpc)
+    STORM_CONFIG="$SUT_HOME"/storm.yaml
+    echo "drpc.servers:" > "$STORM_CONFIG"
+    echo "      "-" "\"`ip route | awk '/default/ {print $3}'`\" >> "$STORM_CONFIG"
```

Comment:
Maybe the command is suitable for Mac OS, but it returns the real IP of Docker host on Linux. I assume we want the IP address on `docker0` network.
```
qxu@ubuntu:~$ ip route
default via 192.168.31.1 dev eth0 
172.17.0.0/16 dev docker0  proto kernel  scope link  src 172.17.0.1 
192.168.31.0/24 dev eth0  proto kernel  scope link  src 192.168.31.200 
qxu@ubuntu:~$ ip route | awk '/default/ {print $3}'
192.168.31.1
```